tests/test_ci_app_host_home_isolation.py

```
@@ -476,11 +481,25 @@ def main() -> int:
         "current-run retry recovery",
     )
-    require(
+    unsafe_signal = re.search(
+        r"(?m)^\s*(?:/bin/)?kill\s+-(?:TERM|KILL|9|15)(?:\s|$)",
+        APP_HOST_PROCESSES,
+    )
+    if unsafe_signal:
+        raise SystemExit(
+            "FAIL: verified app-host cleanup must not signal a reusable PID: "
+            f"{unsafe_signal.group(0).strip()}"
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)